fuzz/afl_targets/graph.rs

```diff
@@ -12,11 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#[macro_use]
+extern crate afl;
```

Comment:
I actually tend to prefer not importing macro and just calling them from their path, but I'm fine anyway,

Reply:
Me too! But I needed to make this change to make it compile, `fuzz` is calling another macro inside it `__fuzz`.

Reply:
ah I see